src/data/constants.js

```diff
@@ -12,6 +12,26 @@ export const ThreadType = {
   DISCUSSION: 'discussion',
 };
 
+/**
+ * Edit actions for posts and comments
+ * @enum {string}
+ */
+export const EditActions = {
```

Comment:
Actually, I had started with a different approach but forgot to rename along the way. Since these are more general actions I had intended to rename this to ContentActions, since some of these may not be related to to editing at all.